pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -208,3 +211,27 @@ func (pr *PipelineRun) GetServiceAccountName(pipelineTaskName string) string {
 	}
 	return serviceAccountName
 }
+
+// PipelineRunTaskSpec  can be used to configure specific
+// specs for a concrete Task
+type PipelineRunTaskSpec struct {
+	TaskName               string       `json:"taskName,omitempty"`
```

Comment:
This is definitely better than just `name` as I had proposed. I think even 'taskName' might be ambiguous and easy to mix-up with a Task CRD name. Consider using pipelineTaskName